mocha.go

```diff
@@ -4,7 +4,7 @@ type mocha struct{}
 
 var _ Theme = mocha{}
 
-func (mocha) Name() string { return "mocha" }
+func (mocha) Name() string { return "Mocha" }
```

Comment:
Same comment as above.
```suggestion
func (mocha) Name() string { return "mocha" }
```